CorePluginFramework/src/au/gov/asd/tac/constellation/plugins/gui/PluginParametersSwingDialog.java

```diff
@@ -234,19 +236,22 @@ public String getResult() {
 
     /**
      * Generates the text of the PluginParameterPane acceptance button based on keywords in the Pane title.
-     * This implementation is useful as it does not require the convoluted work of enabling developers to set 
-     * the acceptance button text at the plugin creation level.
+     * If developers have specified an acceptance button text, this specified text is returned.
+     * If no text was specified a key word is extracted from the title and returned. 
+     * If no word key word is extracted, "OK" is returned
      * 
-     * @param title
      * @return 
      */
     private String getAcceptanceButton() {
+        if (!StringUtils.isBlank(acceptanceText)) {
```

Comment:
`StringUtils.isNotBlank`

Reply:
@antares1470, I resolved this change and mustn't have pushed my commit. Sorry about that.. Should be all good to go once sonar cloud passes :)